ui/pages/Project/Project.jsx

```diff
@@ -45,6 +46,7 @@ class Project extends React.PureComponent {
           <Route path={`${match.url}/analysis_group/:analysisGroupGuid`} component={ProjectPageUI} />
           <Route path={`${match.url}/family_page/:familyGuid/matchmaker_exchange`} component={Matchmaker} />
           <Route path={`${match.url}/family_page/:familyGuid`} component={FamilyPage} />
+          <Route path={`${match.url}/rnaseq_results/:familyGuid/:individualGuid/:hasRnaOutlierData/:hasRnaSpliceOutlierData`} component={RnaSeqResultPage} />
```

Comment:
`familyGuid` should not be in the URL as individuals GUIDs are unique per family. Also, `hasRnaOutlierData` and `hasRnaSpliceOutlierData` do not belong in the URL either, the component should pull that information from the state

Reply:
Okay